src/tests/integration/IntegrationBase.t.sol

```diff
@@ -456,9 +456,9 @@ abstract contract IntegrationBaseTest is DssTest {
         // Rest of the liquidity can be withdrawn
         hub.exec(ilk);
         vow.heal(_min(vat.sin(address(vow)), vat.dai(address(vow))));
-        assertEq(vat.gem(ilk, address(end)), 0);
+        assertRoundingEq(vat.gem(ilk, address(end)), 0);
         assertEq(vat.sin(address(vow)), 0);
-        assertGe(vat.dai(address(vow)), prevDai); // As also probably accrues interest
+        //assertGe(vat.dai(address(vow)), prevDai); // As also probably accrues interest
```

Comment:
why is it commented?

Reply:
failing with metamorpho for the moment

Reply:
There is a rounding error making it off by one. These tests are overly strict imo.

Reply:
alright thx

Reply:
We can replace the whole line by this one (removing the comment as I'm not sure why it mentions that probably accrues interest when the time is static):
```
assertApproxEqAbs(vat.dai(address(vow)), prevDai, roundingTolerance * RAY);
```

Reply:
Same here, replace it by:
```
assertApproxEqAbs(vat.dai(address(vow)), prevDai, roundingTolerance * RAY);
```

Reply:
re-enabled, but relaxed the assert

Reply:
LGTM